src/lib.rs

```diff
@@ -188,31 +188,39 @@
 //! ## Incomparable variants/items
 //!
 //! Similar to the `skip` attribute, with an `incomparable` attribute variants
-//! or items can be skipped from partial compare trait implementations, meaning
-//! they will always yield `false` for equality operations and `None` for
-//! `partial_cmp`.
+//! or items can be skipped from the partial compare trait implementations
+//! [`PartialEq`] and [`PartialOrd`], meaning they will always yield `false` for
+//! equality operations and `None` for `partial_cmp`. This results in all
+//! comparisons but `!=` i.e. `==`, `<`, `<=`, `>=` and `>` with the marked
+//! variant or struct evaluating to `false`.
```

Comment:
```suggestion
//! Similar to the `skip` attribute, `incomparable` can be used to skip variants
//! or items in [`PartialEq`] and [`PartialOrd`] trait implementations, meaning they will always yield `false` for
//! `eq` and `None` for `partial_cmp`. This results in all
//! comparisons but `!=`, i.e. `==`, `<`, `<=`, `>=` and `>`, with the marked
//! variant or struct evaluating to `false`.
```
This probably needs to be formatted to apply the maximum length per line.

Reply:
it would be `false` for `partial_eq`

Reply:
My god I'm so confused, looked up the names again:
`false` for `eq()` and `None` for `partial_cmp()`.

Reply:
Right, `Eq` is just a marker.